tests/ProjectStorageBackendTest.php

```diff
@@ -90,6 +92,47 @@ public function testProjectStorageAssignBackend(string $backendName): void
         $this->client->purgeDeletedProject($project['id']);
     }
 
+    public function testProjectStorageAssignBackendFailedWithNonNumericBackendId(): void
+    {
+        $name = 'My org';
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => $name,
+        ]);
+
+        $project = $this->client->createProject($organization['id'], [
+            'name' => 'My test',
+            'dataRetentionTimeInDays' => 1,
+        ]);
+
+        $apiPostMethod = new ReflectionMethod(ManageClient::class, 'apiPost');
```

Comment:
Tu by  som sa zamyslel ci si toho klienta nevytvorit nejak rucne bokom napriklad tak ako to je tu https://github.com/keboola/kbc-manage-api-php-client/blob/86166a67f3da8b861a47b5a782b1f2d8b0c7f4f3/tests/CommonTest.php#L51-L79, ale obecne proti tomu nic nemam, takto by som to riesil ja, plus by si potom nemusel mat ten phpstan ignor na riadku 114 a 124

Reply:
to je velmi dobrý nápad, jen se nesmí změnit nastavení, které očekáváme od klienta, ale to asi nebude časté